---
title: "Antibiotic resistance determination using <i>Enterococcus faecium</i> whole-genome sequences: a diagnostic accuracy study using genotypic and phenotypic data"
collection: publications
permalink: /publication/2024-01-01-efm-amr-prediction
excerpt: 'This work represents the largest evaluation to date on the accuracy of antibiotic resistance predictions from <i>E. faecium</i> genomes, using a dataset of 4,382 genomes with available culture-based AST phenotypes. I curated a catalogue of 228 genetic markers involved in resistance to 12 different antibiotics in <i>E. faecium</i>, and developed a bioinformatics pipeline to predict antibiotic resistance from E. faecium genomes, with important improvements in the accuracy of predictions. Given the mortality burden of antibiotic-resistant <i>E. faecium</i> and the increasingly routine use of WGS in clinical microbiology labs, these results and resources will facilitate the adoption of WGS as a tool for the diagnosis and surveillance of AMR in <i>E. faecium</i>.'
date: 2024-01-01
venue: 'The Lancet Microbe'
paperurl: 'https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4345808'
citation: '<b>Coll F</b>, Gouliouris T, Blane B, <i>et al</i>. Antibiotic resistance determination using <i>Enterococcus faecium</i> whole-genome sequences: a diagnostic accuracy study using genotypic and phenotypic data. The Lancet Microbe (Accepted).'
---
This work represents the largest evaluation to date on the accuracy of antibiotic resistance predictions from <i>E. faecium</i> genomes, using a dataset of 4,382 genomes with available culture-based AST phenotypes. I curated a catalogue of 228 genetic markers involved in resistance to 12 different antibiotics in <i>E. faecium</i>, and developed a bioinformatics pipeline to predict antibiotic resistance from E. faecium genomes, with important improvements in the accuracy of predictions. Given the mortality burden of antibiotic-resistant <i>E. faecium</i> and the increasingly routine use of WGS in clinical microbiology labs, these results and resources will facilitate the adoption of WGS as a tool for the diagnosis and surveillance of AMR in <i>E. faecium</i>.

Recommended citation: <b>Coll F</b>, Gouliouris T, Blane B, <i>et al</i>. Antibiotic resistance determination using <i>Enterococcus faecium</i> whole-genome sequences: a diagnostic accuracy study using genotypic and phenotypic data. The Lancet Microbe (Accepted).

[Download paper here](http://francesccoll.github.io/files/SSRN-id4345808-1.pdf)
